cumulusci/core/flowrunner.py

```diff
@@ -218,10 +234,11 @@ def __init__(
             callbacks = FlowCallback()
         self.callbacks = callbacks
 
-        # TODO: Support CLI/Runtime Options
-        if not options:
-            options = {}
-        self.runtime_options = options
+        self.runtime_options = defaultdict(dict)
+        if options:
+            for key, value in options.items():
```

Comment:
i sorta feel like this should be in the CLI and flowrunner just takes a nested dict... thoughts?

Reply:
That sounds nice, but I wasn't confident we don't have yaml with options like this.

Reply:
I don't think this codepath was taken for yaml options in baseflow... hmm... eh its fine by me.

Reply:
I think you're right. I moved the __ handling to the CLI